scripts/flow/environment.js

```diff
@@ -14,6 +14,11 @@ declare var __UMD__: boolean;
 declare var __EXPERIMENTAL__: boolean;
 declare var __VARIANT__: boolean;
 
+declare var __FIZZ_INSTRUCTION_SET_COMPLETE_SEGMENT_FUNCTION__: string;
+declare var __FIZZ_INSTRUCTION_SET_COMPLETE_BOUNDARY_FUNCTION__: string;
+declare var __FIZZ_INSTRUCTION_SET_STYLE_INSERTION_FUNCTION__: string;
+declare var __FIZZ_INSTRUCTION_SET_CLIENT_RENDER_FUNCTION__: string;
```

Comment:
Is this required given the var name is embedded in a string in the src? would it be better to encode the exports as `'"function ..."'` and not have the tokens wrapped in single quotes in format config?

Reply:
In this PR that would work fine, but in @mofeiZ's PR, the plan is to inject the implementation directly into the source file. Not as a string. (Meant to unstage this before committing so that was more clear but I forgot.)